base_tier_validation_server_action/tests/test_tier_validation.py

```diff
@@ -0,0 +1,177 @@
+# Copyright 2020 Ecosoft (http://ecosoft.co.th)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+from odoo.tests import common
+from odoo.tests.common import tagged
+
+from .common import setup_test_model, teardown_test_model
+from .tier_validation_tester import TierValidationTester
```

Comment:
Please do not duplicate this files, take them from `odoo.addons.base_tier_validation.tests...`

Reply:
Thank you @LoisRForgeFlow 
I can do for common, but not yet for the test model, as I need more feature. (don't know how to inherit it, by code).
https://github.com/ecosoft-odoo/server-ux/blob/14.0-add-base_tier_validation_action/base_tier_validation_server_action/tests/tier_validation_tester.py#L22-L28

Reply:
I see, I don't know either, it is likely possible but it is also likely not worth it :smile: